### PR TITLE
Touchup rmi manpage

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Touches up a few grammar and formatting issues with the podman rmi man page.

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: docs/podman-rmi.1.md [hunk 1]
```diff
@@ -15,10 +15,11 @@ Removes one or more locally stored images.
 
 **-all**, **-a**
 
-Remove all of the locally storage images
-**--force, -f**
+Remove all of the local storage images.
```
Comment (rhatdan) [inline docs/podman-rmi.1.md]: I still don't like this line.

How about
Remove all images in the local storage.

File: docs/podman-rmi.1.md [hunk 2]
```diff
@@ -15,10 +15,11 @@ Removes one or more locally stored images.
...
 
-Executing this command will stop all containers that are using the image and remove them from the system
+**--force**, **-f**
+
+Executing this command will stop all containers that are using the image and remove them from the system.
```
Comment (rhatdan) [inline docs/podman-rmi.1.md]: "This option will cause `podman` to stop all running containers that are using the image before removing them from the system."

Comment (mheon) [reply docs/podman-rmi.1.md]: Not just stop, remove - we can't leave containers around if they refer to images that no longer exist

Comment (TomSweeneyRedHat) [reply docs/podman-rmi.1.md]: I tweaked both suggestions to:

"This option will cause podman to remove all containers that are using the image before removing the image from the system."

Holler if it needs further tweaking.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**